.travis.yml

```diff
@@ -3,7 +3,7 @@ notifications:
 language: ruby
 script:
 - bundle exec rubocop
-- cd tests && ./test.sh
+- cd tests && ./test.sh && cd ..
```

Comment:
Because the script expects to be run from that directory. This should probably change, but separate from the deploy fix.